src/app/pages/Login/Login.tsx

```diff
@@ -67,9 +67,7 @@ function Login(): JSX.Element {
           <Button variant="secondary">Log in</Button>
 
           {errorMessage && (
-            <div className={styles.registerForm__error}>
-              Error: {errorMessage}
-            </div>
+            <div className={styles.registerForm__error}>{errorMessage}</div>
```

Comment:
Not related to this change, but I noticed that the class is called `registerForm`, but you are in the `Login` page.